<launch>
    <node pkg="cut_planner" type="LaserToImage.py" name="scan_to_img" output="screen">
      <param name="use_pc" value="False"/>
      <param name="image_width" value="100"/>
      <param name="scan_size" value="720"/>
    </node>
    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
      <remap from="image" to="/image_scan"/>
      <param name="autosize" value="true" />
    </node>
</launch>
